app/views/cats/show.html.erb

```diff
@@ -1,15 +1,21 @@
 <div class="container">
- <div><%= image_tag(@cat.picture_url) %></div>
- <div class="cat-title"><p><%= @cat.name %></p></div>
- <p><%= @cat.breed %></p>
- <p><%= @cat.color %></p>
- <p><%= @cat.age %></p>
-
- <div class="details">
-   <h2>A bit more about this cat</h2>
-   <p><%= @cat.description %></p>
- </div>
- <%= link_to 'Bookings', new_cat_booking_path(@cat) %>
+  <div class="booking-form">
```

Comment:
why is the class called "booking-form" when there is no booking form inside?

Reply:
Sandra asked me to add . She will discuss with you about it. Also I have push another edited one ..so please ignore this one plesae.